ext/opcache/jit/zend_jit.c

```diff
@@ -2622,19 +2622,18 @@ static int zend_jit(const zend_op_array *op_array, zend_ssa *ssa, const zend_op
 					/* THROW and EXIT may be used in the middle of BB */
 					/* don't generate code for the rest of BB */
 
-					/* Skip current opline for call_level computation
-					 * Don't include last opline because end of loop already checks call level of last opline */
+					/* Skip current opline for call_level computation. */
 					i++;
-					for (; i < end; i++) {
+					for (; i <= end; i++) {
 						opline = op_array->opcodes + i;
 						if (zend_jit_inc_call_level(opline->opcode)) {
 							call_level++;
 						} else if (zend_jit_dec_call_level(opline->opcode)) {
 							call_level--;
 						}
 					}
-					opline = op_array->opcodes + i;
-					break;
+					opline = op_array->opcodes + end;
```

Comment:
May be you should fix only this single line?
```diff
- opline = op_array->opcodes + i;
+ opline = op_array->opcodes + end;
```

Reply:
alright